Essentials/GridFinder.cs

```diff
@@ -0,0 +1,321 @@
+﻿using Sandbox.Game.Entities;
+using Sandbox.Game.Entities.Character;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using VRage.Game.ModAPI;
+using VRage.Groups;
+using VRageMath;
+
+namespace ALE_Core.Utils {
```

Comment:
The bracket style is inconsistent with the rest of the Torch codebase, I'd also prefer this to be in an Essentials namespace